docs/content/docs/recipes/node.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Using Fluid with NodeJS
+menuPosition: 3
+author: sdeshpande3
+aliases:
+  - "/start/nodejs-tutorial/"
+---
+
+In this tutorial, you'll learn about using the Fluid Framework by building a simple application in NodeJS that enables connected clients to generate random numbers and display the result of any changes to the shared state.  You'll also learn how to connect the Fluid data layer in [Node](https://nodejs.org/).
+
+To jump ahead into the finished demo, check out the [Node demo in our FluidExamples repo](https://github.com/microsoft/FluidExamples/tree/main/node-demo).
+
+The following image shows the random number generated open in four terminals after every one second.
+
+![Four terminals with the Random Number app open in them.](https://user-images.githubusercontent.com/46719950/139499122-76edb832-50ec-4c92-b6b7-06869e9bef9f.png)
```

Comment:
the second terminal from this image has different numbers from the other 3 after like the first 4/5ths of them, and they're not just out of order either

Reply:
thanks for noticing this. I have updated the image. Not sure why the second terminal was not updating the values properly.